demo/monolayer_growth/src/behaviours.h

```diff
@@ -0,0 +1,115 @@
+// -----------------------------------------------------------------------------
+//
+// Copyright (C) 2021 CERN & University of Surrey for the benefit of the
+// BioDynaMo collaboration. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// See the LICENSE file distributed with this work for details.
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// -----------------------------------------------------------------------------
+
+#ifndef BEHAVIOURS_H_
+#define BEHAVIOURS_H_
+
+#include "biodynamo.h"
+#include "core/behavior/behavior.h"
+#include "cycling_cell.h"
+#include "sim_param.h"
+
+namespace bdm {
+
+// Define growth behaviour
+struct Growth : public Behavior {
+  BDM_BEHAVIOR_HEADER(Growth, Behavior, 1);
+
+  Growth() { AlwaysCopyToNew(); }
+  virtual ~Growth() {}
+
+  void Run(Agent* agent) override {
+    if (auto* cell = dynamic_cast<CyclingCell*>(agent)) {
+      auto* random = Simulation::GetActive()->GetRandom();
+      real_t ran = random->Uniform(0, 1) * 1.0;
+      auto* sparam =
+          Simulation::GetActive()
+              ->GetParam()
+              ->Get<SimParam>();  // get a pointer to an instance of SimParam
+
+      // Counter for Delta t at each stage
+      // Used for calculating probability of moving to next state.
+      cell->SetDelt(cell->GetDelt() + 0.1);
```

Comment:
Is the time step hard coded here but can it be changed somewhere else? If it can be changed and we speak of the simulation time step
```
const auto* param = Simulation::GetActive()->GetParam();
cell->SetDelt(cell->GetDelt() + param->simulation_time_step);
```

Reply:
I guess you're right, but let's wait for @BJackal to be on the safe side

Reply:
Delta t (Delt) is hard coded into the cell on its creation but is just set to 0 initially and increments at each time step by the amount of time we wish to increase for cell cycle. In this case 1/10th of an hour. This could be changed to be either a member of the agent being initialised (not sure on this as its bad practice for when you have more than one agent) or we can set the simulation time step to determine this. E.g Our current time steps are in tenth of time steps (3100 time steps = 310 hours). The second option would require a change to the code though as currently we want to set this value back to zero when we step into a new state (As it is essentialy counting the number of time steps the cell has existed in this state againt the maximum amount it is allowed to).